src/main/java/org/webshar/hrms/model/db/RolePermission.java

```diff
@@ -0,0 +1,73 @@
+package org.webshar.hrms.model.db;
+
+import java.io.Serializable;
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.stereotype.Component;
+
+@Component
+@Entity
+@Table(name = "role_permission", uniqueConstraints = @UniqueConstraint(columnNames = {"id"}))
+public class RolePermission implements Serializable
```

Comment:
Considering one resource can have many permissions, I think instead of creating a new entity here, modify the Role entity to include List<Permission> to model a one-to-many relationship. 
e.g.
class Role{
@OnetoMany
List<Permission> permissions
} 
Explore this option.
JPA will create a join table in this case with role id and permission id columns.